daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto

```diff
@@ -435,11 +435,11 @@ enum BuiltinFunction {
 
   ADD_NUMERIC = 107;    // *Available in versions >= 1.7*
   SUB_NUMERIC = 108;    // *Available in versions >= 1.7*
-  MUL_NUMERIC = 109;    // *Available in versions >= 1.7*
-  DIV_NUMERIC = 110;    // *Available in versions >= 1.7*
+  MUL_NUMERIC_LEGACY = 109;    // *Available in versions >= 1.7*
```

Comment:
Probably worth mentioning when this became "legacy".

Reply:
In an upcoming PR